src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -450,6 +523,28 @@ impl Node {
             eval out.hover ((t) action_bar.set_visibility(t));
 
 
+            // === Set Node Error ===
+
+            eval frp.set_error_status([model,error_anim](error) {
+                model.set_error(error);
+                match error.as_ref() {
+                    Some(error) => {
+                        let error_data:visualization::Data = error.clone().into();
+                        model.visualization.frp.set_data.emit(error_data);
+
+                        let red:color::Lcha = color::Rgba::red().into();
+                        error_anim.target.emit(red);
```

Comment:
you are doing 2 frp emmits from these block, which means that this could be split into more fine-grained FRP endpoints. Please do it. Emitting more than one thing is almost always a bad design. Moreover, then you can not emit things at all, and use our new syntax here, like `error_anim.target <+ something`.